### PR TITLE
[vcpkg] Bump baseline

### PR DESCRIPTION
**Pull request by m-kuhn**
Pull in some updates (among these: Qt 6.8)

### CODE REVIEW

**Comment by m-kuhn** [comment]
CC @troopa81 I think you have been waiting for this

**Comment by troopa81** [comment]
Yes! I was actually building it on my computer 😅 But is there PyQt 6.8.0? I see no release. Or is it built with a previous version of pyqt?

**Comment by m-kuhn** [comment]
It tries to build with a previous version, let's see if this succeeds

**Comment by m-kuhn** [comment]
@troopa81 builds are there (without pyqt, if required we could use dev versions)

**Comment by troopa81** [comment]
@m-kuhn I failed to use generated binaries. I just click the qgis.exe, the splashcreen appear and nothing. Is there anything else to do?

**Comment by nicogodet** [comment]
I confirm there is something wrong with this build

**Comment by m-kuhn** [comment]
Unsure if it's related to Qt 6.8 or missing bindings and I'm currently lacking the time to look deeper

**Comment by troopa81** [comment]
> Unsure if it's related to Qt 6.8 or missing bindings and I'm currently lacking the time to look deeper

I tried also another qt5 on another PR and it fails. There is maybe more...

**Comment by nicogodet** [comment]
Windows Qt6 build from master works on my side

**Comment by troopa81** [comment]
I'm building it on debug to check what happens

**Comment by nicogodet** [comment]
Artifact still crashes
Looks related to Qt 6.8

**Comment by m-kuhn** [comment]
thanks for testing, I enabled console output, hopefully there's something interesting

**Comment by troopa81** [comment]
I figured out what's the issue here see https://github.com/microsoft/vcpkg/issues/42661

I managed to build a working version of QGIS with Qt 6.8.0. 

I have a [patch](https://github.com/troopa81/QGIS/commit/1655e8a121093e178c3864e9bd7bc49b8acc8f5b#diff-e5d5111297d1c878b8c128ba5cd6e260189259b0d720e8ed44a64375517692b6L74) but do we want to push that or wait for vcpkg/Qt to fix upstream?

**Comment by troopa81** [comment]
> I guess we are not in a hurry to merge this, it was mostly for experimenting. I'd rather work with upstream and bump after. Do you know if the upcoming 6.8.1 update is also affected?

No I'll try to test it.

**Comment by troopa81** [comment]
@m-kuhn Actuall, I think this is fixed in 6.8.1 (see [my comment](https://github.com/microsoft/vcpkg/issues/42661#issuecomment-2538921570) ). So I'm gonna wait for 6.8.1 which is already approved

**Comment by nicogodet** [comment]
Note: PyQt is now 6.8.0
https://pypi.org/project/PyQt6/

File: src/app/CMakeLists.txt [hunk 1]
```diff
@@ -530,7 +530,7 @@ else()
       set(MACOSX_BUNDLE "MACOSX_BUNDLE")
     endif()
   endif()
-  add_executable(${QGIS_APP_NAME} ${MACOSX_BUNDLE} WIN32 ${QGIS_APPMAIN_SRCS})
+  add_executable(${QGIS_APP_NAME} ${MACOSX_BUNDLE} ${QGIS_APPMAIN_SRCS})
```
Comment (m-kuhn) [inline src/app/CMakeLists.txt:533]: TODO

**Comment by m-kuhn** [comment]
Hmm... looks like we need to wait for this to be merged

**Review by nirvn — APPROVED** [review]

**Comment by nirvn** [comment]
I can't get enough of these vcpkg reports :) nice cleanup, approved.

**Pull request merged**